extension/data.ts

```diff
@@ -86,6 +86,27 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+const WORD_DICT_ENTRY_REGEX = /^(.+?)\s+(?:\[(.*?)\])?\s*\/(.+)\//;
```

Comment:
I plan on changing this regex to be more readable with named capturing groups, but for now I'm keeping it the same as it was before to reduce the amount of change in this pull request.